src/pages/_app.tsx

```diff
@@ -16,11 +17,45 @@ if (!BASE_URL) {
 
 const App = ({ Component, pageProps }) => {
   const { push, pathname: pathName, asPath, locale } = useRouter();
+  const storedDarkModeString = process.browser
+    ? window.localStorage.getItem("darkMode")
+    : true;
+  const storedLocaleString = process.browser
+    ? window.localStorage.getItem("locale")
+    : locale;
+
+  const storedDarkMode = Boolean(storedDarkModeString === "true");
```

Comment:
Doesn't `===` already return a boolean?

Reply:
Yes I forgot to remove the Boolean stuff 😄